activity_calendar/templates/activity_calendar/activity_moment_form_page.html

```diff
@@ -91,7 +91,7 @@ <h4>Edit occurence data</h4>
         it will copy the attribute from the general activity instead.
     </p>
 
-    {% render_form has_submit_button=False id="submit_form" method="post" %}
+    {% render_form has_submit_button=False id="submit_form" method="post" uses_file_field=True %}
```

Comment:
Clearable file-inputs don't really work with the current form layout

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/34304046/132732629-6a20d3bb-220a-497f-9cf0-d200ebcd7b20.png)
</details>